src/sentry/sentry_metrics/querying/api.py

```diff
@@ -185,44 +200,68 @@ def __init__(
         self._query = query
         self._group_bys = group_bys
 
-    def _parse_query(self) -> Optional[Sequence[Condition]]:
+        # We want to sanitize the input in order to avoid any injection attacks due to the string interpolation that
```

Comment:
I wanted to reduce the likelihood of people finding combinations of chars in filters/group bys strings that can lead to a different query being generated since they are merged together into a final MQL query.